processes/security_release_process.md

```diff
@@ -0,0 +1,77 @@
+# Security Release Process
+
+The security release process covers the steps required to plan/implement
+a security release.
+
+The steps include:
+
+* Get agreement on the list of vulnerabilities to be addressed
+  and the planned date for the releases. This is done in an issue
+  in the private security repo titled `Next Security Release`
+
+* Once agreement on the list and date has been agreed, validate
+  that all vulnerabilities have been assigned a CVE following
+  the [cve_management_process](https://github.com/nodejs/security-wg/blob/master/processes/cve_management_process.md).
+
+* Co-ordinate with the Release team members to line up one
+  or more releasers to do the releases on the agreed date.
```

Comment:
Given https://github.com/nodejs/node/pull/21275 there should probably be an explicit step to notify the Build WG to ensure coverage to mitigate any infrastructure issues around the agreed date. 

cc @nodejs/build

Reply:
+1 to what @richardlau said, can we also add a line saying that while the release is going out, the releaser must be around in node-build in case issues come up

Reply:
done